plugin/system_copy.vim

```diff
@@ -85,7 +85,11 @@ function! s:PasteCommandForCurrentOS()
   elseif os == s:windows
     return 'paste'
   elseif os == s:linux
-    return 'xsel --clipboard --output'
+    if !empty($WAYLAND_DISPLAY)
+      return 'wl-copy'
```

Comment:
Just to confirm, it looks like you've implemented this in the "PasteCommand" section, although it looks to me like a copy command. Likewise, is there an equivalent for the copy side (or paste side if this is in fact in the wrong section)?

Reply:
argh yes you are right sorry. I implemented this workaround in my vimrc and only thought to search for `xsel` to fix it. Will update.

Are there any tests in this project I should be updating or would it be just a manul test?

Reply:
Updated. Should be correct now :)